admin/tools/docker/1_build-latest-image.sh

```diff
@@ -84,28 +86,26 @@ DEPS_DIR=$(echo $DEPS_DIR | sed 's%\(.*[^/]\)/*%\1%')
 # So, copy it from templates to SCRIPT_DIR directory
 SCRIPT_DIR="$DOCKERDIR/scripts"
 
-. "$DOCKERDIR"/dist.sh
-TPL_DEPS_SCRIPT="$DEPS_DIR/qserv-install-deps-"$DIST".sh"
+TPL_DEPS_SCRIPT="$DEPS_DIR/qserv-install-deps-debian8.sh"
 
 printf "Add physical link to dependencies install script: %s\n" "$TPL_DEPS_SCRIPT"
 ln -f "$TPL_DEPS_SCRIPT" "$SCRIPT_DIR/install-deps.sh"
 
 # Build the release image
-TAG="fjammes/qserv:$VERSION"
+TAG="qserv/qserv:$VERSION"
 printf "Building latest release image %s from %s\n" "$TAG" "$DOCKERDIR"
-docker build --tag="$TAG" "$DOCKERDIR"
+docker build $CACHE_OPT --tag="$TAG" "$DOCKERDIR"
```

Comment:
More or less, all variables should be quoted in a shell script.  I highly recommend the usage of http://www.shellcheck.net/, I have it integrated with vim to lint while I code.